charts/komodor-agent/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v1
 name: komodor-agent
 description: Watches and sends kubernetes resource-related events
 icon: https://raw.githubusercontent.com/komodorio/helm-charts/master/k8s-watcher.svg
-version: 2.0.5
+version: 0.0.0
```

Comment:
Is this a mistake? or for testing?

Reply:
Not a mistake, for local dev it will be like that,
the chart version is not managed anymore via commits.
It will now be managed using git tags and when publishing the version we will calculate the next version based on previous tags

Reply:
Hmmm ok. It does mean the generated readme uses `0.0.0` as its version, not sure if its critical or not.

Reply:
I will remove it from the readme